### PR TITLE
ISPN-5621 Remove add and addAll support from entrySet methods.

### PR DESCRIPTION
**Pull request by wburns**
https://issues.jboss.org/browse/ISPN-5621

### CODE REVIEW

File: core/src/test/java/org/infinispan/api/APINonTxTest.java [hunk 1]
```diff
@@ -8,6 +8,9 @@
 import org.infinispan.test.SingleCacheManagerTest;
 import org.infinispan.test.fwk.TestCacheManagerFactory;
 import org.testng.annotations.Test;
+
+import javax.resource.NotSupportedException;
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/api/APINonTxTest.java]: I think this import is not used.

Comment (wburns) [inline core/src/test/java/org/infinispan/api/APINonTxTest.java]: Yep :) auto imported the wrong exception and missed removing, doh.

**Comment by wburns** [comment]
Refactored imports.

**Comment by pruivo** [comment]
integrated! thanks @wburns :)

**Pull request closed without merging**